influxql/parser.go

```diff
@@ -94,7 +94,7 @@ func (p *Parser) ParseStatement() (Statement, error) {
 	case ALTER:
 		return p.parseAlterStatement()
 	case SET:
-		return p.parseSetStatement()
```

Comment:
I changed the name of the function to match the pattern we use for all the other `parse...` functions.